partials/schema-prop.html

```diff
@@ -1,7 +1,8 @@
 {% from "./type.html" import type %}
 {% from "./schema-item.html" import schemaItem %}
+{% from "./description.html" import getDescription %}
```

Comment:
@anbreaker this is the part I didn't cover in the video, so I do not do all 😄 

as you can see, file `description.html` doesn't exist, you need to create it, and put in there a macro called `getDescription` that will return such a div element `<div class="text-sm markdown">{{ description | markdown2html | safe }}</div>` with proper value of course. More about macros -> https://mozilla.github.io/nunjucks/templating.html#macro

good luck, and remember to test to make sure it works before you let me know to have another look